implement basic export history functionality

cadobson opened the pull request:
I'm seeking feedback about this implementation. I added an rpc method, called that rpc method when a new button is pushed, and used vscode's built-in file save feature.

dustinlacewell commented:
The implementation looks good to me.

I think you accidentally added `apps/spellbound.json` though.

cadobson commented:
Very good. I will remove that file, and try to format the output to be more human readable.

cadobson commented:
Removed erroneous file, made the output text more human readable, and cleaned up the look of the button. If we're going to add a Stop🛑 button  in the future, it occurs to me that the UI is getting crowded and that right beside the Send button is perhaps not the best place for the Export button. For the time being, I'm happy to put this forward for consideration.
